addons/sale_timesheet/tests/test_product_template.py

```diff
@@ -0,0 +1,39 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.exceptions import ValidationError
+from odoo.addons.sale_timesheet.tests.common import TestCommonSaleTimesheet
+from odoo.tests import tagged
+
+
+@tagged('-at_install', 'post_install')
```

Comment:
```suggestion
```

It can be `at_install`, imo

Reply:
I used the common setup from the module, which requires it. I can write a specific setup for this test class if needed.